user-interface/src/main/java/life/qbic/datamanager/views/layouts/BoxLayout.java

```diff
@@ -1,182 +0,0 @@
-package life.qbic.datamanager.views.layouts;
-
-import com.vaadin.flow.component.Component;
-import com.vaadin.flow.component.Text;
-import com.vaadin.flow.component.button.Button;
-import com.vaadin.flow.component.html.H2;
-import com.vaadin.flow.component.html.Span;
-import com.vaadin.flow.component.orderedlayout.FlexComponent;
-import com.vaadin.flow.component.orderedlayout.VerticalLayout;
-import life.qbic.datamanager.views.notifications.DisplayMessage;
-
-/**
- * <b>Box Layout</b>
- *
- * <p>A box with a shadow containing a title, description, a layout for fields, a layout for buttons
- * and a span to add links.
- * Furthermore, the description text can be toggled visible or invisible
- *
- * @since 1.0.0
- */
-public class BoxLayout extends VerticalLayout {
```

Comment:
How is this change related to the issue? What happened here and why?

Reply:
The BoxLayout was a remnant from the time in which we relied on the Vaadin specified styling(Vertical Layout). This change aims to harmonize on how the login, registration and password related components can be harmonized in our style format. (CSS based styling)